setools/policyrep/selinuxpolicy.pxi

```diff
@@ -351,12 +351,35 @@ cdef class SELinuxPolicy:
     #
     def lookup_boolean(self, name):
         """Look up a Boolean."""
+        new_name = self.check_boolean_sub(name)
+
         for b in self.bools():
-            if b == name:
+            if b == new_name:
                 return b
 
         raise InvalidBoolean("{0} is not a valid Boolean".format(name))
 
+    def check_boolean_sub(self, name):
```

Comment:
This should go as a standalone function in `util.pxi`, preferrably with a name like `lookup_boolean_name_sub()`, since this is not directly tied to the policy.

Reply:
Thank you, updated.